cpp/src/fil/fil.cu

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, NVIDIA CORPORATION.
+ * Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
nitpicking here, this should be 2019-2020

Reply:
Done